### PR TITLE
ci: add swagger editor validate

### PR DESCRIPTION
**Pull request by ariwk**
### Proposed changes

Due to problems in OpenAPI specifications automated checks must be implemented #279

### Checklist

Before creating a PR, run through this checklist and mark each as complete:
- [x] I have read the [`CONTRIBUTING`](CONTRIBUTING.md) document
- [x] If applicable, I have added tests that prove my fix is effective or that my feature works
- [x] If applicable, I have checked that any relevant tests pass after adding my changes
- [x] I have updated any relevant documentation

### CODE REVIEW

**Comment by ariwk** [comment]
Hi @SchweizerischeBundesbahnen/sbb-clew-polarion-maintainers 
we need to first fix the problems in the spec before merging ;-) and rename/adapt/document the workflow
Cheers, Adam

**Review by grigoriev — APPROVED** [review]

**Pull request merged**
